zulipterminal/config/keys.py

```diff
@@ -198,7 +198,7 @@
     }),
     ('AUTOCOMPLETE', {
         'keys': {'ctrl f'},
-        'help_text': 'Autocomplete @mentions',
+        'help_text': 'Autocomplete @mentions and #stream_names',
```

Comment:
You could include this file again in the commit title in this later commit, or at least mention the keys/README change in the summary.